events/meeting_handler.py

```
@@ -0,0 +1,57 @@
+        self._leave()
+
+    async def _decision(self):
+        await self._broadcast_message({"handler": "meeting", "type": "decision"})
+
+    async def _start(self):
+        await self._broadcast_message({"handler": "meeting", "type": "start"})
+
+    async def _end(self):
+        await self._broadcast_message({"handler": "meeting", "type": "end"})
+
+    async def _broadcast_message(self, message):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_broadcast_message`

Add return type annotation: `None`

(ANN202)